packages/grainIntegration-merkle/README.md

```diff
@@ -0,0 +1,40 @@
+# Merkle Distribution Grain Integration
+
+#### ⚠️ This contract is still under active development ⚠️
+
+## How It Works
+
+`MerkleReem` is heavily based on [work from balancer](https://github.com/balancer-labs/erc20-redeemable)
+and is a scalable way to distribute rewards in the form of an ERC20 on an EVM
+blockchain. There are 3 roles:
+
+- `DEFAULT-ADMIN`: Assigned to the deployer of the contract. This role
+  is responsible for managing the below two roles
+- `SEEDER_ROLE`: Manually assigned and managed by the `DEFAULT_ADMIN`
+  role. Responsible for publishing distributions on chain using the
+  `seedAllocations` function.
+- `PAUSER_ROLE`: Manually assigned and managed by the `DEFAULT_ADMIN`
```

Comment:
You still have some "pauser"s here and in the contract that need changed to "delayer"

Reply:
Thanks! that's a bad miss for me